zeppelin-web/package.json

```diff
@@ -93,7 +93,7 @@
     "postcss-loader": "^3.0.0",
     "protractor": "^5.4.1",
     "raw-loader": "^0.5.1",
-    "rimraf": "^3.0.2",
+    "rimraf": "^2.5.4",
```

Comment:
Hi. I have a question about this pr.
Can you tell me why you downgraded the version of `rimraf`?

Reply:
Oh? I missed this thing. I also wonder why.

Reply:
Oh I'm so sorry.. it's 3.0.2 version on my local but it's changed somehow, I don't know why.. May be there was a mistake when I make pr. Should I revert this pr??

Reply:
> Oh I'm so sorry.. it's 3.0.2 version on my local but it's changed somehow, I don't know why.. May be there was a mistake when I made PR. Should I revert this pr??

I only updated rimraf version to 3.0.2 on master branch, but not on my branch 'remove_sap'. There was mistake when I made PR. If I should revert this pr, please let me know and  I'll do it.

Reply:
I think we should change this. Because, the version was updated in pr [#4414](https://github.com/apache/zeppelin/pull/4414)